ironfish/src/network/peers/peer.ts

```diff
@@ -412,6 +412,10 @@ export class Peer {
     return this.state.identity
   }
 
+  getIdentity(): Identity | null {
+    return this.state.identity
+  }
```

Comment:
what do you think about changing this to `get identity` instead? that way you just have to do `peer.identity` to access it instead of `peer.getIdentity()`

Reply:
I'd suggest to make this simpler we just remove `getIdentity()` and use `peer.state.identity` instead, like we do in the rest of the codebase. I'm open to adding `getIdentity` (or an `identity` property), but I think it'd be better to do a whole-code refactor to use it

Reply:
I think i'd prefer to keep as is too out of all the options so things are more consistent